### DISCUSSION TITLE
Cannot find location of files in windows

### DISCUSSION THREAD

**Discussion by cybercoder-naj** [microsoft/WSL#7095] [Q&A]
I installed WSL2 and Ubuntu 20.04 LTS. I used the `.appx` file and extracted it. The terminal executable file is located in `D:\Ubuntu20\Ubuntu\ubunt2004.exe` and _not_ in the `C:\` drive. I don't know where the files that I create in this WSL are store in windows. How do I find them?

**Comment by therealkenc** [comment]
Your WSL-side files are available on the Windows-side via the `\\wsl$\` UNC path. If you are looking for the vhdx, that is in `%LocalAppData%\Packages\CanonicalGroupLimited[...]`.
